rules/klayout/drc/testing/run_regression.py

```diff
@@ -32,28 +32,30 @@
     --run_name=<run_name>               Select your run name.
 """
 
+from subprocess import check_call
+
 from docopt import docopt
 import os
 import datetime
 import xml.etree.ElementTree as ET
 import csv
 import time
 import re
-
-from sympy import arg
+import pandas as pd
+import logging
 
```

Comment:
consider using https://docs.pytest.org/en/7.2.x/

Reply:
@proppy That's a big change right now. We can move to it later.

Reply:
then can we file an issue to track that work?

Reply:
@proppy Yes please do.

Reply:
https://github.com/google/globalfoundries-pdk-libs-gf180mcu_fd_pr/issues/79